src/Pim/Bundle/CatalogBundle/Elasticsearch/Filter/TextFilter.php

```diff
@@ -139,4 +164,18 @@ protected function checkValue(AttributeInterface $attribute, $value)
             throw InvalidPropertyTypeException::stringExpected($attribute->getCode(), static::class, $value);
         }
     }
+
+    /**
+     * Escapes value
+     *
+     * @param string $value
+     *
+     * @return string
+     */
+    protected function escapeValue($value)
+    {
+        $regex = '#[-+=|! &(){}\[\]^"~*<>?:/\\\]#';
```

Comment:
Sometimes we do, sometimes we don't, sometimes it's not even the query but just the fact that we use the '.raw' version of the field. That's why we split those two (+ those query may change after the benchmark).